README.md

```diff
@@ -1,16 +1,45 @@
 [![Build Status](https://travis-ci.org/mixpanel/mixpanel-iphone.svg?branch=yolo-travis-ci)](https://travis-ci.org/mixpanel/mixpanel-iphone)
 [![Cocoapods Version](http://img.shields.io/cocoapods/v/Mixpanel.svg?style=flat)](https://mixpanel.com)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![Apache License](http://img.shields.io/cocoapods/l/Mixpanel.svg?style=flat)](https://mixpanel.com)
 
 **Want to Contribute?**
 
 The Mixpanel library for iOS is an open source project, and we'd love to see your contributions! We'd also love for you to come and work with us! Check out http://boards.greenhouse.io/mixpanel/jobs/25226#.U_4JXEhORKU for details.
 
-# Painless Installation
+# Painless Installation (CocoaPods)
 
 Mixpanel supports `Cocoapods` for easy installation.
 To Install, see our **[full documentation »](https://mixpanel.com/help/reference/ios)**
 
+# Carthage
```

Comment:
So this is pretty in depth, whereas the cocoapods one is just like "we do cocoapods!, click here for details" 

Maybe we should have inline cocoapods instructions here too, so they are all in one place. But the tradeoff of adding all this stuff inline is that the README starts looking like you need a CS degree to add Mixpanel to your app. In the shoes of an iOS app dev, what are your thoughts? What are some iOS projects with particularly good READMEs that you like?